.github/workflows/pull_request.yml

```
@@ -0,0 +1,32 @@
+name: Unit Tests
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [18.x] # You can specify the Node.js versions you want to test against
```

```suggestion
        node-version: [18.x] 
```
what is the recommended node version? also to keep that in the docs to running locally and in prod the same

---

done, and updated - thanks!